packages/headless/src/controllers/commerce/field-suggestions/headless-category-field-suggestions.ts

```diff
@@ -113,7 +136,16 @@ export function buildCategoryFieldSuggestions(
 
 function loadFieldSuggestionsReducers(
   engine: CommerceEngine
-): engine is CommerceEngine {
-  engine.addReducers({commerceFacetSet});
+): engine is CommerceEngine<
+  FieldSuggestionsOrderSection &
+    CommerceFacetSetSection &
+    CategoryFacetSearchSection &
+    FacetSearchSection
+> {
+  engine.addReducers({
+    fieldSuggestionsOrder,
+    commerceFacetSet,
```

Comment:
I wonder why we need both... I think we should only need categoryFacetSearchSet

Reply:
Removed in https://github.com/coveo/ui-kit/pull/4245/commits/4f48a8912ab1fe5bb164276048e06f110eb4833c since it's not needed!

Reply:
I've re-added it actually, since it's used when building category field suggestions facet search requests https://github.com/coveo/ui-kit/blob/a1aba3e05763a123e7fdadf99318601887df9a0d/packages/headless/src/features/commerce/facets/facet-search-set/category/commerce-category-facet-search-request-builder.ts#L18